Modules/BiddingManager/GUI.lua

```diff
@@ -289,23 +289,29 @@ function BiddingManagerGUI:GenerateAuctionOptions()
             width = 0.43,
             order = 6
         },
-        cancel = {
-            name = CLM.L["Cancel"],
-            desc = CLM.L["Cancel your bid."],
-            type = "execute",
-            func = (function() BiddingManager:CancelBid() end),
-            width = 0.43,
-            order = 8
-        },
         pass = {
             name = CLM.L["Pass"],
             desc = CLM.L["Notify that you are passing on the item. Cancels any existing bids."],
             type = "execute",
             func = (function() BiddingManager:NotifyPass() end),
+            disabled = (function()
+				return CLM.CONSTANTS.AUCTION_TYPES_OPEN[self.auctionType] and BiddingManager:GetLastBidValue() > 0
+			end),
             width = 0.43,
             order = 9
         }
     }
+    if not CLM.CONSTANTS.AUCTION_TYPES_OPEN[self.auctionType] then
```

Comment:
Please do not remove the button as it will modify the size (we would need to resize window dynamically all the time). Please use `disabled = (function() return CLM.CONSTANTS.AUCTION_TYPES_OPEN[self.auctionType] end)` instead (remember that at that scope you need to have the `self.auctionType` available).